soh/soh/Enhancements/randomizer/3drando/item_pool.cpp

```diff
@@ -554,6 +554,42 @@ static void PlaceVanillaBossKeys() {
   }
 }
 
+static void PlaceVanillaBeehiveRupees() {
+  PlaceItemInLocation(RC_KF_STORMS_GROTTO_BEEHIVE_LEFT, BLUE_RUPEE, false, true);
```

Comment:
All these are using the `RC_` enums instead of the 3ds rando keys.hpp enums, which is causing item placement failure when shuffle beehives is off due to the these enums marking the wrong locations as placed.